src/js/components/Notification/Notification.js

```diff
@@ -45,38 +94,74 @@ const Notification = ({ message, onClose, id, status, title, toast }) => {
   ]);
 
   const { icon: CloseIcon } = theme.notification.close;
-  const { icon: StatusIcon, color } = theme.notification[status];
+  const { icon: StatusIcon, background, color } = theme.notification[status];
   const { color: closeIconColor } = theme.notification.close;
 
+  const { direction, truncate } = toast
```

Comment:
will we fall back to base values if not specified under `theme.notification.toast`?

Reply:
Yes we will, that is what is seen under https://deploy-preview-5958--sad-tereshkova-173d07.netlify.app/?path=/story/visualizations-notification-status--status -- since base.js doesn't define `theme.notification.toast.direction` or t`heme.notification.toast.truncate`.